.github/workflows/ci.yaml

```diff
@@ -67,7 +67,7 @@ jobs:
       - name: Check dependency licenses
         env:
           GITHUB_TOKEN: ${{ github.token }} # Note: this is required for licensei auth in steps to avoid rate-limiting.
-        run: make license-check
+        run: go mod vendor && make license-check
```

Comment:
vending in deps in CI before checking their licenses per https://github.com/banzaicloud/koperator/pull/853#discussion_r961651920 to resolve spurious license errors in CI when contacting github.com, thanks to @panyuenlau for this tip.